<!--  -->
<launch>
  <node pkg="mrop" type="reference_generator" name="reference_generator" output="screen">
    <param name="parent_frame"           value="world"/>
    <param name="child_frame"            value="reference"/>
    
    <param name="reference_generator_active" value="true"/>
    <param name="trajectory_paused"      value="false"/>
    <param name="trajectory_stopped"     value="false"/>

    <param name="loop_rate"              value="100.0"/>    

    <param name="trajectory_type"        value="0"/>

    <param name="initial_x"              value="0.0"/>
    <param name="initial_y"              value="0.0"/>
    <param name="initial_theta"          value="0.0"/>
    <param name="linear_velocity"        value="0.1"/>
    <param name="harmonic_period"        value="5.0"/>
    <param name="harmonic_radius_x"      value="1.0"/>
    <param name="harmonic_radius_y"      value="1.0"/>
    <param name="harmonic_multiplier_x"  value="1.0"/>
    <param name="harmonic_multiplier_y"  value="1.0"/>
  </node>
</launch>

